src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -885,6 +890,18 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                 .EndControl());
         }
 
+        private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Automation_ProgressRecord()
+        {
+            yield return new FormatViewDefinition(
+                "ProgressRecord",
+                CustomControl
+                    .Create(outOfBand: true)
+                        .StartEntry()
+                            .AddScriptBlockExpressionBinding($@"'{FormatStrings.ProgressFormatString}' -f ""$(if ($_.PercentComplete -ge 0) {{'{{0,3}}% - ' -f $_.PercentComplete}})$($_.Activity)$(if ($_.StatusDescription -ne '{FormatStrings.Processing}') {{"": $($_.StatusDescription)""}})$(if ($_.CurrentOperation) {{"" ($($_.CurrentOperation))""}})""")
```

Comment:
Please add a screenshot with the output.

Reply:
Will add. Thanks.